ui/sortable.js

```diff
@@ -12,6 +12,8 @@
 //>>description: Enables items in a list to be sorted using the mouse.
 //>>docs: http://api.jqueryui.com/sortable/
 //>>demos: http://jqueryui.com/sortable/
+//>>css.structure: ../themes/base/sortable.css
```

Comment:
Out of scope of this PR, but would be good to figure out: Both `draggable.css` and `sortable.css` are there simpy to define `touch-action: none;` for handles. We could move this to `core.css` as a generic class. @tjvantoll @scottgonzalez thoughts?